src/bugsnag.js

```diff
@@ -115,6 +118,12 @@
   // Notify Bugsnag about an error by passing in a `name` and `message`,
   // without requiring an exception.
   self.notify = function (name, message, metaData, severity) {
+    if (!name) {
+      name = "UnspecifiedBugsnagError";
```

Comment:
Might be worth pulling this to the top of the file where we declare some other constant-esque variables?